sessionctx/stmtctx/stmtctx.go

```diff
@@ -79,6 +79,7 @@ type StatementContext struct {
 	BatchCheck             bool
 	InNullRejectCheck      bool
 	AllowInvalidDate       bool
+	CanRuntimePrune        bool
```

Comment:
It looks weird to me in the PR for this field. Since it's useless in anyplace.

Reply:
emmm~ I had removed it and pick it to following PR :D